src/app/layouts/stepper/stepper-linear-sample/stepper-linear-sample.component.html

```diff
@@ -0,0 +1,103 @@
+<div class="sample-split">
+    <igx-buttongroup [values]="modes" (selected)="toggleModes($event)">
+    </igx-buttongroup>
+</div>
+
+<igx-stepper #stepper [linear]="linear">
+
+    <ng-template igxStepActiveIndicator>
+        <igx-icon>edit</igx-icon>
+    </ng-template>
+
+    <igx-step #step1 [isValid]="form1.form.valid">
+        <span igxStepTitle>Personal Info</span>
+        <div igxStepContent>
+            <form #form1="ngForm">
+                <igx-input-group>
+                    <input igxInput name="fullName" type="text" [(ngModel)]="user.fullName" required />
+                    <label igxLabel for="fullName" id="fullName">Full Name</label>
+                </igx-input-group>
+                <igx-input-group>
+                    <input igxInput name="email" type="email" [(ngModel)]="user.email" required email />
+                    <label igxLabel for="email" id="email">Email</label>
+                </igx-input-group>
+            </form>
+            <div class="sample-step-actions">
+                <button igxButton="raised" (click)="stepper.next()"
+                    [disabled]="!form1.form.valid && stepper.linear">NEXT</button>
+            </div>
+        </div>
+    </igx-step>
+
+    <igx-step #step2 [isValid]="form2.form.valid">
+        <span igxStepTitle>Delivery address</span>
+        <div igxStepContent>
+            <form #form2="ngForm">
+                <igx-input-group>
+                    <input igxInput name="city" type="text" [(ngModel)]="user.city" required />
+                    <label igxLabel for="city" id="city">City</label>
+                </igx-input-group>
+                <igx-input-group>
+                    <input igxInput name="street" type="text" [(ngModel)]="user.street" required />
+                    <label igxLabel for="street" id="street">Street</label>
+                </igx-input-group>
+            </form>
+            <div class="sample-step-actions">
+                <button igxButton="raised" (click)="stepper.prev()">Previous</button>
+                <button igxButton="raised" (click)="stepper.next()"
+                    [disabled]="!form2.form.valid && stepper.linear">Next</button>
+                <button igxButton (click)="stepper.navigateTo(3)"
+                    [disabled]="!form2.form.valid && stepper.linear">Skip</button>
```

Comment:
Not a big fan of this button here because it's not obvious to the end user that it will skip the next optional step. Since it's present on Step No 2 I immediately assume that it will skip the current step and not the following. Might want to rename the button to something more descriptive or remove it entirely.

Reply:
@YoannaIvanova do you have any suggestions? 
https://cloud.indigo.design/share/run/76temq3fbt7k?open_toc=true#f=stepper%20orientation%20horizontal_linear%20step%203

Reply:
I remember we decided to remove the Skip button.